plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -36,9 +37,34 @@ public static String generateSchema() {
                     "    },");
         }
         schemaString.append("},\n");
-        Set configObjects = new LinkedHashSet<>(ConfigurationAsCode.get().getConfigurators());
-        System.out.println(configObjects.size());
-        Iterator<Objects> obj = configObjects.iterator();
+
+
+        /**
+         * This generates the schema for the base configurators
+         * Iterates over the base configurators and adds them to the schema.
+         */
+
+        String output = "{";
```

Comment:
This was required for the beautification else it was complaining of invalid schema.Will remove it later.